spec/collator.html

```diff
@@ -114,7 +114,7 @@ <h1>Intl.Collator.supportedLocalesOf ( _locales_ [ , _options_ ] )</h1>
       </emu-alg>
 
       <p>
-        The value of the `length` property of the *supportedLocalesOf* method is 1.
+        The value of the *"length"* property of the *supportedLocalesOf* method is 1.
```

Comment:
Looks like `*supportedLocalesOf*` → <code>\`supportedLocalesOf\`</code> got changed in a few spots and missed in a few spots. (Here's one of the missed spots.)

Reply:
I'll make a new PR for these, good eye!